packages/vats/src/vat-bank.js

```diff
@@ -361,6 +368,22 @@ export function buildRootObject() {
               proposedName,
             }),
           );
+
+          if (nameAdmin) {
+            // publish settled issuer identity
```

Comment:
please include this explanation of conditioned behavior in the jsdoc too (e.g. line 111)